docs/developing-locally-docker.rst

```diff
@@ -200,3 +199,96 @@ Prerequisites:
+
+#.  **Let's Encrypt**
+    
+    The official line from Let’s Encrypt is: 
+
+    [For local development section] ... The best option: Generate your own certificate, either self-signed or signed by a local root, and trust it in your operating system’s trust store. Then use that certificate in your local web server. See below for details. 
+
+    See `letsencrypt.org - certificates-for-localhost`_
+
+    .. _`letsencrypt.org - certificates-for-localhost`: https://letsencrypt.org/docs/certificates-for-localhost/
+
+#.  **mkcert: Valid Https Certificates For Localhost**
+    
+    `mkcert`_ is a simple by design tool that hides all the arcane knowledge required to generate valid TLS certificates. It works for any hostname or IP, including localhost. It supports macOS, Linux, and Windows, and Firefox, Chrome and Java. It even works on mobile devices with a couple manual steps.
+
+    See https://blog.filippo.io/mkcert-valid-https-certificates-for-localhost/
+
+    .. _`mkcert`:  https://github.com/FiloSottile/mkcert/blob/master/README.md#supported-root-stores
+
+After installing a trusted TLS certificate, configure your docker installation. We are going to configure an ``nginx`` reverse-proxy server. This makes sure that it does not interfere with our ``traefik`` configuration that is reserved for production environements.
+
+These are the places that you should configure to secure your local environment.
+
+certs
+~~~~~
+
+Take the certificates that you generated and place them in a folder called ``certs`` on the projects root folder. Assuming that you registered your local hostname as ``my-dev-env.local``, the certificates you will put in the folder should have the names ``my-dev-env.local.crt`` and ``my-dev-env.local.key``.
+
+local.yml
+~~~~~~~~~
+
+#. Add the ``nginx-proxy`` service. ::
+
+    ...
+
+    nginx-proxy:
+      image: jwilder/nginx-proxy:alpine
+      container_name: nginx-proxy
+      ports:
+        - "80:80"
+        - "443:443"
+      volumes:
+        - /var/run/docker.sock:/tmp/docker.sock:ro
+        - ./certs:/etc/nginx/certs
+      restart: always
+      depends_on:
+        - django
+    
+    ...
+
+#. Link the ``nginx-proxy`` to ``django`` through environmental variables.
+   
+   ``django`` already has an ``.env`` file connected to it. Add the following variables. You should do this especially if you are working with a team and you want to keep your local environment details to yourself.
+
+   ::
+
+      # HTTPS
+      # ------------------------------------------------------------------------------
+      VIRTUAL_HOST=my-dev-env.local
+      VIRTUAL_PORT=8000
+
+   The services run behind the reverse proxy.
+
+config/settings/local.py
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You should allow the new hostname. ::
+
+  ALLOWED_HOSTS = ["localhost", "0.0.0.0", "127.0.0.1", "my-dev-env.local"]
+
+Rebuild your ``docker`` application. ::
+
+  $ docker-compose -f local.yml up -d --build
+
+Go to your browser and type in your usrl, ``
```

Comment:
Looks like something is missing here... Did you mean to write something like that?

```suggestion
Go to your browser and type in your URL bar ``https://my-dev-env.local``
```